logstash-filter-dissect.gemspec

```diff
@@ -1,8 +1,8 @@
 Gem::Specification.new do |s|
-  s.name = 'logstash-filter-example'
-  s.version = '1.0.0'
+  s.name = 'logstash-filter-dissect'
+  s.version = '1.0.5'
   s.licenses = ['Apache License (2.0)']
-  s.summary = "This example filter replaces the contents of the message field with the specified value."
+  s.summary = "This dissect filter will destructurize text in multiple fields."
```

Comment:
"destructurize text in multiple fields" is ... confusing to me. Maybe the gem summary isn't important, though.